src/overlays/actors/ovl_En_Door/z_en_door.c

```diff
@@ -15,7 +29,112 @@ void EnDoor_Destroy(Actor* thisx, GlobalContext* globalCtx);
+void func_80866F94(EnDoor*, GlobalContext*);
+void func_80867080(EnDoor*, GlobalContext*);
+void func_80867144(EnDoor*, GlobalContext*);
+void func_808670F0(EnDoor*, GlobalContext*);
+void func_80866A5C(EnDoor*, GlobalContext*);
+
+// I have no idea what this is
+s32 D_808675D0[] = {
+    0x0C00030E, 0x02060017, 0x00080200, 0x00060002, 0x09070500,
+};
+
+s32 D_808675E4[] = {
+    0x0F090047, 0x0F0C0015, 0x0C000113, 0x11000F0D, 0x000A0F0F, 0x00070F00, 0x00020909,
+    0x0509080C, 0x00021900, 0x1C08070F, 0x0D000209, 0x09050F0D, 0x00EA0F11, 0x00E70F00,
+    0x00E20909, 0x0F0D00DC, 0x0F0D00D9, 0x0F120001, 0x05090909, 0x09000000,
+};
+
+s32 D_80867634[] = { 0x02060017, 0x0002091C, 0x05000000 };
+
+s32 D_80867640[] = { 0x02150016, 0x00010509, 0x0B000000 };
+s32 D_8086764C[] = { 0x02160005, 0x0002090A, 0x05000000 };
+s32 D_80867658[] = { 0x0C00020F, 0x001C0802, 0x090C020F, 0x0A160002, 0x090C050C, 0x0003110F,
+                     0x0D000D00, 0x33080209, 0x0C0F1600, 0x02090C05, 0x090C0000 };
+s32 D_80867684 = 0x05000000;
+
+s32 D_80867688[] = { 0x0F160002, 0x090F0500 };
+
+s32 D_80867690[] = { 0x0F160002, 0x09100500 };
+
+s32 D_80867698[] = { 0x0F160002, 0x09110500 };
+
+s32 D_808676A0[] = {
+    0x020A0015, 0x000F0216, 0x00050002, 0x09120802, 0x02092209, 0x21050000,
+};
+
+s32 D_808676B8[] = {
+    0x0800140C, 0x0003070F, 0x08000105, 0x09130208, 0x00141E02, 0x09130505,
+};
+
+s32 D_808676D0 = 0x05000000;
+
+s32 D_808676D4 = 0x05000000;
+
+s32 D_808676D8[] = {
+    0x0C000305,
+    0x0F120001,
+    0x05091400,
+};
+
+s32 D_808676E4[] = {
+    0x08000209,
+    0x16050000,
+};
+
+s32 D_808676EC = 0x05000000;
+
+s32 D_808676F0[] = {
+    0x0C000305, 0x0F0A0007, 0x05020A00, 0x1400F909, 0x15000000,
+};
+
+s32 D_80867704 = 0x05000000;
+s32 D_80867708 = 0x05000000;
+s32 D_8086770C = 0x05000000;
+s32 D_80867710 = 0x05000000;
+
+s32 D_80867714[] = {
+    0x0F140002,
+    0x09170500,
+};
+
+s32 D_8086771C[] = {
+    0x0C000107, 0x0F021E02, 0x0918050C, 0x0003FB0F, 0x1400F709, 0x0C000000,
+};
+
+s32 D_80867734[] = {
+    0x0F140002,
+    0x09190500,
+};
+
+s32 D_8086773C[] = {
+    0x0F140002,
+    0x091A0500,
+};
+
+s32 D_80867744[] = {
+    0x0C000204, 0x00638006, 0x0F140008, 0x091B0F12, 0x0002091B, 0x05000000,
+};
+
+s32 D_8086775C[] = {
+    0x00342017, 0x004B2013, 0x000E0402, 0x091D003B, 0x0108003D, 0x0202091E, 0x091F0505,
+};
+
+s32 D_80867778 = 0x09200000;
+s32 D_8086777C = 0x05000000;
+s32 D_80867780 = 0x05000000;
+s32 D_80867784[] = { 0x0F160002, 0x09230500 };
+
+s32 D_8086778C[] = {
+    &D_808675D0, &D_808675E4, &D_80867634, &D_80867640, &D_8086764C, &D_80867658, &D_80867684, &D_80867688,
+    &D_80867690, &D_80867698, &D_808676A0, &D_808676B8, &D_808676D0, &D_808676D4, &D_808676D8, &D_808676E4,
+    &D_808676EC, &D_808676F0, &D_80867704, &D_80867708, &D_8086770C, &D_80867710, &D_80867714, &D_8086771C,
+    &D_80867734, &D_8086773C, &D_80867744, &D_8086775C, &D_80867778, &D_8086777C, &D_80867780, &D_80867784,
+};
```

Comment:
Taking another look at this. 
This is only used by `func_80133038` which is the main function from `code_801323D0` (named z_schedule by #509 ). This function is meant to take an array, so passing a `s32` is wrong. 
Could you please fix this?